VndbSharp/VndbUtils.cs

```diff
@@ -0,0 +1,87 @@
+﻿using System;
+using System.Collections.Generic;
+using System.IO;
+using System.IO.Compression;
+using System.Net;
+using System.Text;
+using System.Threading.Tasks;
+using Newtonsoft.Json;
+using VndbSharp.Structs.Models.Dumps;
+
+namespace VndbSharp
+{
+    public class VndbUtils
+    {
+        protected DateTime LastTime = DateTime.Now;
+        protected static Boolean FirstRun = true;
```

Comment:
It appears that you don't quite understand what i mean, by having the DateTime be null by default, it acts as the FirstRun boolean, then it becomes the Last Run DateTime value itself, so the variable is being used for both at once.

It really doesn't matter how the FirstRun is checked in the long run, especially since i'm not sure how Rate Limiting will be done in the long run, because if yorhel wants a long period between uses i'll need to figure out a cross platform way of storing the last call, and what it was for, which means it's entirely plausible this approach won't be viable :(